tests/FixerDefinition/VersionSpecificCodeSampleTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\FixerDefinition;
+
+use PhpCsFixer\FixerDefinition\VersionSpecificCodeSample;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ */
+final class VersionSpecificCodeSampleTest extends \PHPUnit_Framework_TestCase
+{
+    public function testConstructorSetsValues()
+    {
+        $code = '<php echo $foo;';
+        $configuration = array(
+            'foo' => 'bar',
+        );
+
+        $codeSample = new VersionSpecificCodeSample(
+            $code,
+            $configuration
+        );
+
+        $this->assertSame($code, $codeSample->getCode());
+        $this->assertSame($configuration, $codeSample->getConfiguration());
+    }
+
+    public function testConfigurationDefaultsToNull()
+    {
+        $codeSample = new VersionSpecificCodeSample('<php echo $foo;');
+
+        $this->assertNull($codeSample->getConfiguration());
+    }
+
+    public function testIsSuitableForDefaultsToTrue()
+    {
+        $version = 9000;
+
+        $codeSample = new VersionSpecificCodeSample(
+            '<php echo $foo;',
+            null
+        );
+
+        $this->assertTrue($codeSample->isSuitableFor($version));
+    }
+
+    /**
+     * @dataProvider providerIsSuitableForVersionUsesVersionSpecification
+     *
+     * @param bool $isSatisfied
+     */
+    public function testIsSuitableForUsesVersionSpecification($isSatisfied)
+    {
+        $version = 9000;
```

Comment:
should be part of data provider.
data provider that provide only expected value, but no input values, looks weird for me